docs/topics/class-based-views/generic-display.txt

```diff
@@ -119,7 +120,8 @@ Finally hook that view into your urls::
 
     # urls.py
     from django.urls import path
-    from books.views import PublisherListView
+
+    from .views import PublisherListView
```

Comment:
Leave alone. This looks like a root URLConf since it uses "publishers/".

Reply:
ok